### PR TITLE
Fix /5/ urls

### PR DESCRIPTION
**Pull request by ksuess**
I referenced a label in training that will be introduced with training PR [#706](https://github.com/plone/training/pull/706)

### CODE REVIEW

**Comment by ksuess** [comment]
Let's wait for the broken/unsecure link…

**Comment by stevepiercy** [comment]
> I referenced a label in training that will be introduced with training PR #706

Did you mean this PR? https://github.com/plone/training/pull/706

**Comment by stevepiercy** [comment]
> Let's wait for the broken/unsecure link…

Strange, it works fine when I visit it. Perhaps temporary? I restarted the test job.

**Review by stevepiercy — APPROVED** [review]
Thank you!

File: docs/backend/upgrading/version-specific-migration/p4x-to-p5x-upgrade.md [hunk 1]
```diff
@@ -448,7 +448,7 @@ The value you use for this CSS rule should identify one of the fontello icons in
 
 It is not possible at this time to set an icon for your add-on package control panels without including CSS in your package.
 
-For documentation on how to use it in your own add-ons see https://training.plone.org/5/mastering-plone-5/registry.html.
+For documentation on how to use it in your own add-ons see {ref}`training:plone5-registry-controlpanel-label`.
```
Comment (stevepiercy) [inline docs/backend/upgrading/version-specific-migration/p4x-to-p5x-upgrade.md]: Actually this label does not resolve.

https://deploy-preview-1463--6-docs-plone-org.netlify.app/backend/upgrading/version-specific-migration/p4x-to-p5x-upgrade.html#control-panel

What should it be?

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: docs/backend/upgrading/version-specific-migration/p4x-to-p5x-upgrade.md (hunk 1 above)
Comment (stevepiercy) [reply docs/backend/upgrading/version-specific-migration/p4x-to-p5x-upgrade.md]: Perhaps this?

```
{doc}`training:mastering-plone-5/registry`
```

**Comment by stevepiercy** [comment]
The URL is still broken according to `linkcheckbroken`: https://github.com/plone/documentation/actions/runs/4382423217/jobs/7671936971#step:5:837

I'm not sure what's the problem. Should we notify the site manager? https://vhs-ehrenamtsportal.de/ (Also their favicon is Pastanaga's.)

**Comment by davisagli** [comment]
> I'm not sure what's the problem. Should we notify the site manager? https://vhs-ehrenamtsportal.de/ (Also their favicon is Pastanaga's.)

This site was deployed to a new server yesterday. Maybe a DNS propagation issue? I'd suggest trying the build again now.

**Pull request merged**